src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,116 @@
+package duke.task;
+
+import org.json.simple.JSONObject;
+
+import duke.date.Date;
+import duke.exception.BadInputFormatException;
+import duke.exception.InvalidDateException;
+
+/** Represents an Event object. */
+public class Event extends DatedTask {
+    /**
+     * Event constructor.
+     *
+     * @param description The event's description
```

Comment:
Just a reminder to add punctuation for these parameter descriptions :)

Reply:
👍🏼